Description: Fix glib header include
 glib headers must be included via glib.h only (not via separate partial
 glib*.h headers). This patch fixes that.
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/665521

Index: dia-0.97.2/objects/custom/shape_typeinfo.c
===================================================================
--- dia-0.97.2.orig/objects/custom/shape_typeinfo.c	2011-12-31 16:43:25.000000000 +0100
+++ dia-0.97.2/objects/custom/shape_typeinfo.c	2012-03-24 22:04:20.000000000 +0100
@@ -27,7 +27,7 @@
 #include "custom_util.h"
 #include <string.h>
 #include <stdarg.h>
-#include <glib/gstrfuncs.h>
+#include <glib.h>
 #include <glib/gstdio.h>
 #include <libxml/parser.h>
 
Index: dia-0.97.2/tests/test-boundingbox.c
===================================================================
--- dia-0.97.2.orig/tests/test-boundingbox.c	2011-12-31 16:43:25.000000000 +0100
+++ dia-0.97.2/tests/test-boundingbox.c	2012-03-24 22:36:02.000000000 +0100
@@ -28,9 +28,6 @@
 #include <glib.h>
 #include <glib-object.h>
 
-#if GLIB_CHECK_VERSION(2,16,0)
-#include <glib/gtestutils.h>
-#endif
 #include "dialib.h"
 
 /*
Index: dia-0.97.2/tests/test-objects.c
===================================================================
--- dia-0.97.2.orig/tests/test-objects.c	2012-03-24 22:50:25.000000000 +0100
+++ dia-0.97.2/tests/test-objects.c	2012-03-24 22:51:01.000000000 +0100
@@ -28,10 +28,6 @@
 #include <glib.h>
 #include <glib-object.h>
 
-#if GLIB_CHECK_VERSION(2,16,0)
-#include <glib/gtestutils.h>
-#endif
-
 #include "object.h"
 #include "plug-ins.h"
 #include "dialib.h"
